OpenSim/Sandbox/ExampleHopperDevice/exampleHopperDevice.cpp

```diff
@@ -168,12 +168,28 @@ void addDeviceConsoleReporterToModel(Model& model, Device& device,
 } // namespace OpenSim
 
 
+void executeExample();
```

Comment:
Renaming `main()` to `executeExample()` requires corresponding modifications to [README.md](https://github.com/opensim-org/opensim-core/blob/master/OpenSim/Sandbox/ExampleHopperDevice/README.md) (bottom), exampleHopperDevice.cpp ([line 35](https://github.com/opensim-org/opensim-core/blob/master/OpenSim/Sandbox/ExampleHopperDevice/exampleHopperDevice.cpp#L35)), and exampleHopperDevice_answers.cpp ([line 35](https://github.com/opensim-org/opensim-core/blob/master/OpenSim/Sandbox/ExampleHopperDevice/exampleHopperDevice_answers.cpp#L35)).